### PR TITLE
Custom toolbar height

### PR DESCRIPTION
**Pull request by mathcarignani**
As we want to customize the input toolbar we need to expose a method to update the height.

### CODE REVIEW

File: JSQMessagesViewController/Controllers/JSQMessagesViewController+Private.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
+//  JSQMessagesViewController+Private.h
+//  Pods
```
Comment (gfzabarino) [inline JSQMessagesViewController/Controllers/JSQMessagesViewController+Private.h]: This shouldn't be Pods :)

**Comment by gfzabarino** [comment]
I added a small comment, other than that we should be good to go.

**Comment by mathcarignani** [comment]
@gfzabarino Done, thanks

**Comment by gfzabarino** [comment]
:+1:

**Pull request merged**